doc/source/abstractions/app-interface.rst

```diff
@@ -61,45 +61,49 @@ within this method.
 
 .. code:: python
 
-    def create_open_region(self, frequency="1GHz", boundary="Radiation",
-                           apply_infinite_gp=False, gp_axis="-z"):
-       """Create an open region on the active editor.
-
-       Parameters
-       ----------
-       frequency : str, optional
-           Frequency with units. The  default is ``"1GHz"``.
-       boundary : str, optional
-           Type of the boundary. The default is ``"Radiation"``.
-       apply_infinite_gp : bool, optional
-           Whether to apply an infinite ground plane. The default is ``False``.
-       gp_axis : str, optional
-           The default is ``"-z"``.
```

Comment:
This needs to have a description added. Also, do you think the method description should end in "on the active editor" or "in the active editor"?